src/IO.cpp

```diff
@@ -217,26 +227,18 @@ ObservationsForModels ReadModelObservations(const std::string& configDir, Utilit
     (*log) << "Observations For Models:" << std::endl;
 
     const std::string scot_data_file = configDir + "/scot_data.csv";
-    if (!Utilities::fileExists(scot_data_file)) throw IOException(scot_data_file + ": File not found!");
-    
     const std::string scot_ages_file = configDir + "/scot_age.csv";
-    if (!Utilities::fileExists(scot_ages_file)) throw IOException(scot_ages_file + ": File not found!");
-
     const std::string waifw_norm_file = configDir + "/waifw_norm.csv";
-    if (!Utilities::fileExists(waifw_norm_file)) throw IOException(waifw_norm_file + ": File not found!");
-
     const std::string waifw_home_file = configDir + "/waifw_home.csv";
-    if (!Utilities::fileExists(waifw_home_file)) throw IOException(waifw_home_file + ": File not found!");
-
     const std::string waifw_sdist_file = configDir + "/waifw_sdist.csv";
-    if (!Utilities::fileExists(waifw_sdist_file)) throw IOException(waifw_sdist_file + ": File not found!");
-
     const std::string cfr_byage_file = configDir + "/cfr_byage.csv";
-    if (!Utilities::fileExists(cfr_byage_file)) throw IOException(cfr_byage_file + ": File not found!");
-
     const std::string scot_frail_file = configDir + "/scot_frail.csv";
-    if (!Utilities::fileExists(scot_frail_file)) throw IOException(scot_frail_file + ": File not found!");
+    const std::string settings_file = configDir + "/parameters.ini";
 
+    int nHealthBoards = ReadNumberFromFile<int>("nHealthBoards", "Settings", settings_file);
```

Comment:
Since there is some duplication here, I'd suggest creating a separate function whose job it is to read in the validation parameters e.g.

```
ValidationParameters ReadValidationParameters(configDir)
{
    ValidationParameters params;
    const std::string settings_file = configDir + "/parameters.ini";

    params.nHealthBoards = ReadNumberFromFile<int>("nHealthBoards", "Settings", settings_file);
    ....
}
```

and then call it from the other I/O functions as necessary.
```